app/templates/main/map.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+
+<script type="text/javascript">
+    function initializeMapAndMarkers() {
+        markerWrappers = [];
+        var minDate = new Date(2015, 0, 1);
+        var bounds = new google.maps.LatLngBounds();
+        {% for report in incident_reports %}
+        $(function() {
+            var marker = new google.maps.Marker({
+                // Use latitude and longitude values from incident report
+                // to set position of marker
+                position:{lat: parseFloat('{{ report.location.latitude }}'), lng: parseFloat('{{ report.location.longitude }}')}
+            });
+            var dateString = '{{ report.date }}';
+            year = parseInt((dateString.split(' '))[0].split('-')[0]);
+            month = parseInt((dateString.split(' '))[0].split('-')[1])-1;
+            day = parseInt((dateString.split(' '))[0].split('-')[2]);
+            incidentDate = new Date(year, month, day);
+            if (incidentDate.getTime() < minDate.getTime())
+                minDate.setTime(incidentDate.getTime());
+            //marker.setMap(map);
+            var contentString = '<div id="content">' +
+                                '<div id="siteNotice">' +
+                                '</div>' +
+                                '<h1 id="firstHeading" class="firstHeading">Vehicle ID: ' + '{{ report.vehicle_id }}' +'</h1>' +
+                                '<div id="bodyContent">' +
+                                '<p>License Plate: ' + '{{ report.license_plate }}' + '</p>' +
+                                '<p>Date: ' + '{{ report.date }}' + '</p>' +
+                                '<p>Duration: ' + '{{ report.duration }}' + '</p>' +
+                                '<p>Agency: ' + '{{ report.agency.name }}' + '</p>' +
+                                '<p>Link to Picture: ' + '{{ report.picture_url }}' + '</p>' +
+                                '<p>Description: ' + '{{ report.description }}' + '</p>' +
+                                '</div>' +
+                                '</div>';
+            bounds.extend(marker.getPosition());
+            markerWrapper = new MarkerWrapper(marker, incidentDate, contentString);
+            markerWrappers.push(markerWrapper);
+            var infoWindow = new google.maps.InfoWindow({
+                content: contentString
+            });
+
+            // Add click listener to marker for displaying infoWindow
+            marker.addListener('click', function() {
+                infoWindow.open(map, marker);
+            });
+        });
+        {% endfor %}
+        var mapProp = {
+                mapTypeId: google.maps.MapTypeId.ROADMAP,
+                mapTypeControl: false,
+                streetViewControl: false
+        };
+        map=new google.maps.Map($('#googleMap')[0],mapProp);
+
+        // Use HTML geolocation to center map if possible
+        if (navigator.geolocation) {
```

Comment:
Do we want to do this by default or just when they select the geolocate button?

@kongsally